### PR TITLE
test: remove useless work for jest

### PR DESCRIPTION
**Pull request by Haroenv**
1. make sure jest env is node
2. remove rewire, and simply export / mock

### CODE REVIEW

File: src/__tests__/changelog.test.js [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-// Mock 'got' requests.
-// To fetch from network, just comment the code below.
-// If you are fetching from network, the tests may timeout. To prevent this,
-// use jest.setTimeout() to increase the async timeout.
-__RewireAPI__.__Rewire__(
-  'got',
-  url =>
-    gotSnapshotUrls.has(url) ? Promise.resolve({ url }) : Promise.reject() // eslint-disable-line prefer-promise-reject-errors
-);
+import { getChangelogs, baseUrlMap } from '../changelog.js'; // eslint-disable-line import/named
```
Comment (Haroenv) [inline src/__tests__/changelog.test.js]:
```suggestion
import { getChangelogs, baseUrlMap } from '../changelog.js';
```

**Comment by bodinsamuel** [comment]
Thanks I was not sure it was needed ^^

File: .babelrc [hunk 2]
```diff
@@ -1,17 +1,12 @@
 {
-  "presets": [
+  presets: [
     [
-      "@babel/env",
+      '@babel/env',
```
Comment (bodinsamuel) [inline .babelrc]: Maybe it's missing a prettier conf somewhere, my editor should have changed that quote too last time

Comment (Haroenv) [reply .babelrc]: yes, it's just prettier, the last time we changed this file it formatted it as json, now json5, not too big of a deal

**Pull request merged**